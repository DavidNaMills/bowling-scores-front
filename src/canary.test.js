import React from 'react';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';

const TestApp = () =>(
    <div>
        <p>Test Component!!!</p>
    </div>
)

describe('canary test', ()=>{
    it('basic test', ()=>{
        expect(true).toBeTruthy();
    });

    test('Canary snapshot test', ()=>{
        const component = renderer.create(<TestApp/>);
        const app = component.toJSON();
        expect(app).toMatchSnapshot();
    });

    it('enzyme canary test', ()=>{
        const app = shallow(<TestApp/>);
        expect(app.find('p').length).toEqual(1);
    })
})