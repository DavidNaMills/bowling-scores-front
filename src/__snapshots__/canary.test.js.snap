// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`canary test Canary snapshot test 1`] = `
<div>
  <p>
    Test Component!!!
  </p>
</div>
`;
